magento/magento2
# Improve StoreRepository exception message

Improve StoreRepository exception message.

### Description (*)
When you run app:config:import with a store that does not exist you will receive a generic error message. 
`Import failed: The store that was requested wasn't found. Verify the store and try again.`

Then you need to start debugging to find out which store id/code causes an error.
This is why I added the id/code to the exception message - to make a developers life easier.

You will not receive a message like:
`Import failed: The store with code be_en that was requested wasn't found. Verify the store and try again.`

### Manual testing scenarios (*)
1. Run `bin/magento app:config:import` with a config store value that does not exist/is not valid
2. Error message is shown


### Contribution checklist (*)
